controllers/alerts/alerts.go

```diff
@@ -7,75 +7,37 @@ package alerts
 
 import (
 	"context"
-	"errors"
-	"fmt"
-	"os"
 	"reflect"
-	"strings"
 
 	"github.com/go-logr/logr"
-	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"k8s.io/utils/ptr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/rules"
 	hcoutil "github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
 )
 
 const (
-	alertRuleGroup                = "kubevirt.hyperconverged.rules"
-	outOfBandUpdateAlert          = "KubeVirtCRModified"
-	unsafeModificationAlert       = "UnsupportedHCOModification"
-	installationNotCompletedAlert = "HCOInstallationIncomplete"
-	singleStackIPv6Alert          = "SingleStackIPv6Unsupported"
-	severityAlertLabelKey         = "severity"
-	healthImpactAlertLabelKey     = "operator_health_impact"
-	partOfAlertLabelKey           = "kubernetes_operator_part_of"
-	partOfAlertLabelValue         = "kubevirt"
-	componentAlertLabelKey        = "kubernetes_operator_component"
-	componentAlertLabelValue      = "hyperconverged-cluster-operator"
-	ruleName                      = hcoutil.HyperConvergedName + "-prometheus-rule"
-	defaultRunbookURLTemplate     = "https://kubevirt.io/monitoring/runbooks/%s"
-	runbookURLTemplateEnv         = "RUNBOOK_URL_TEMPLATE"
+	ruleName                  = hcoutil.HyperConvergedName + "-prometheus-rule"
+	defaultRunbookURLTemplate = "https://kubevirt.io/monitoring/runbooks/%s"
```

Comment:
Why is this still here?

Reply:
The pr is still draft

Reply:
I didn't do finals touch yet

Reply:
but actually all are used in metrics-test.go

Reply:
all are used in metrics_test.go